lib/formats/csv/csv.lingo-template

```diff
@@ -0,0 +1,10 @@
+csv = row:r1 (nl row)*:rs { consArray(:r1, :rs) };
+row = cell:c1 (sp cell)*:cs { consArray(:c1, :cs) }; 
+cell = ws* escaped$s1 ws* { unescapeQ($s1) } | plain$s2 { $s2 };
+escaped = (q (!q char)* q)+;
+plain = (!(nl | sp | q) char)*;
+char = '\u0000'-'\uFFFF';
```

Comment:
Range better ends with \uFFFD, since FFFF and FFFE considered invalid characters, and the latter also used to detect byte order.